### PR TITLE
[Merged by Bors] - doc(FieldTheory/SeparableDegree): fix the formula in docstring

### PR DESCRIPTION
**Pull request by acmepjz**
---

The formula in https://leanprover-community.github.io/mathlib4_docs/Mathlib/FieldTheory/SeparableDegree.html is broken. I hope this will fix it, but I can't find a way to preview.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by alreadydone** [comment]
Preview looks good: https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%22https%3A%2F%2Fgithub.com%2Fleanprover-community%2Fmathlib4%2Fblob%2Facmepjz_sep_degree_docstring%2FMathlib%2FFieldTheory%2FSeparableDegree.lean%22#docs

File: Mathlib/FieldTheory/SeparableDegree.lean [hunk 1]
```diff
@@ -25,9 +25,9 @@ This file contains basics about the separable degree of a field extension.
 
   **Remark:** if `E / F` is not algebraic, then this definition makes no mathematical sense,
   and if it is infinite, then its cardinality doesn't behave as expected (namely, not equal to the
-  field extension degree of `separableClosure F E / F`). For example,
-  $\operatorname{Emb}_{\mathbb{Q}}(\mathbb{Q}(\mu_{p^\infty}))\cong\mathbb{Z}_p^\times$ which is
-  uncountable, while $[\mathbb{Q}(\mu_{p^\infty}):\mathbb{Q}]$ is countable.
+  field extension degree of `separableClosure F E / F`). For example, if $ F = \mathbb{Q} $ and
+  $ E = \mathbb{Q}( \mu_{p^\infty} ) $, then $ \operatorname{Emb}_F (E) $ is isomorphic to
```
Comment (alreadydone) [inline Mathlib/FieldTheory/SeparableDegree.lean:29]:
```suggestion
  $ E = \mathbb{Q}( \mu_{p^\infty} ) $, then $ \operatorname{Emb}_F (E) $ is in bijection with
  $\operatorname{Gal}(F/E)$, which is isomorphic to
```

**Comment by alreadydone** [comment]
Thanks!

maintainer merge

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by alreadydone** [comment]
I realized the [observablehq.com](https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%22https%3A%2F%2Fgithub.com%2Fleanprover-community%2Fmathlib4%2Fblob%2Facmepjz_sep_degree_docstring%2FMathlib%2FFieldTheory%2FSeparableDegree.lean%22#docs) link I used for preview is for doc-gen3, not doc-gen4, so we're not really sure if the fix works ... The online docs haven't been rebuilt yet.

**Pull request closed without merging**